scripts/install.sh

```
@@ -119,6 +127,56 @@ fi
+        done
+      fi
+    fi
+
+    if [[ "${CUDA_RUNTIME}" == "true" ]]; then
+      BACKEND="cuda"
+    else
+      BACKEND="vulkan"
+    fi
+    log "Auto-selected ${BACKEND} inference backend."
+  fi
+elif [[ "${BACKEND}" != "auto" ]]; then
```

Shellcheck (0.11.0)

[warning] In POSIX sh, [[ ]] is undefined.

(SC3010)